.github/actions/ipsec-key-rotate/action.yaml

```diff
@@ -3,19 +3,13 @@ description: Rotates keys and checks that established connections are not interr
 inputs:
```

Comment:
*(Regarding commit description.)*

Nit: We don't use `feat:`. See the commit history of that directory for the proper keyword(s).

Reply:
Renamed to `workflows`. Thanks!